.github/pull_request_template.md

```diff
@@ -6,10 +6,8 @@
  
 ### Check List
 - [ ] New functionality includes testing.
-  - [ ] All tests pass
 - [ ] New functionality has been documented.
   - [ ] New functionality has javadoc added
-- [ ] Commits are signed per the DCO using --signoff 
```

Comment:
Perhaps we should keep this, but rename it:

> Commits are signed with a real name per the DCO

The GitHub Action does not check for real names.